packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1601,7 +1594,7 @@ module.exports = function(log, config, oauthdb) {
     return this.send(
       Object.assign({}, message, {
         headers,
-        subject: gettext('Low recovery codes remaining'),
+        subject: gettext('Low Recovery Codes Remaining'),
```

Comment:
I know you didn't specify the text, `Low` doesn't seem like the right word here. It seems like `Few` would be more suitable, or `Create More Recovery Codes to Avoid Lockout`

Reply:
This is related to the behavioural change that I mentioned in https://github.com/mozilla/fxa/pull/2017#issuecomment-517201294.

I've compromised, so this now reads `%(numberRemaining)s Recovery Codes Remaining` instead.